content/guides/concepts/hashes.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Hashes"
+menu:
+    guides:
+        parent: concepts
+---
+
+Hashes are functions that take some arbitrary input and return a fixed-size alphanumeric string. The appearance of that alphanumeric string depends on the given hash algorithm in use, such as [SHA-1](https://en.wikipedia.org/wiki/SHA-1) (used by Git), [SHA-256](https://en.wikipedia.org/wiki/SHA-2), or [BLAKE2](https://en.wikipedia.org/wiki/BLAKE_(hash_function)#BLAKE2). Have a look at the [full list of hash functions](https://en.wikipedia.org/wiki/List_of_hash_functions) for more.
+
+As an example, the input of the string:
+
+```
+Hello world
+```
+
+would be represented by **SHA-1** as:
+
+```
+0x7B502C3A1F48C8609AE212CDFB639DEE39673F5E
+```
+
+However, exactly the same input generates the following output by **SHA-256**:
+
+```
+0x64EC88CA00B268E5BA1A35678A1B5316D212F4F366B2477232534A8AECA37F3C
+```
+
+Notice that the second hash is longer than the first one. This is because SHA-1 creates a 160 bit hash, while SHA-256 creates a 256 bit hash. Also, the prepended `0x` is just and indicator that tells us that the following digest is represented as a hexadecimal number.
```

Comment:
> People should generally know these are the same thing, but it’s a lot easier in general (and especially for non-native english speakers) to stick with the same terminology throughout.

I have to admit, I just recently realised that base 16 is hexadecimal because I've never seen that wording being used like that in this context before. So I'll go with `base 16 (or hexadecimal)` here.